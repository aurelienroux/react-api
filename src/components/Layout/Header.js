import React from "react";
import { Link } from "react-router-dom";

const Header = () => {
  return (
    <div>
      <h2>Header</h2>
      <Link to="/">Home </Link>
      <Link to="/sites">| Sites </Link>
      <Link to="/clients"> | Clients </Link>
      <hr />
    </div>
  );
};

export default Header;
