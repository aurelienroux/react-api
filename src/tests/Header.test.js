import React from "react";
import Header from "../components/Layout/Header";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";

it("renders component without crashing", () => {
  render(
    <Router>
      <Header />
    </Router>
  );
});
