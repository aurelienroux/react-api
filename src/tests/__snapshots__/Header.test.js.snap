// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative makeStyles-menu-2 MuiAppBar-colorPrimary"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <button
      aria-label="menu"
      className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
      </span>
    </button>
    <h6
      className="MuiTypography-root MuiTypography-h6"
    >
      <a
        className="home-link"
        href="/"
        onClick={[Function]}
      >
        Scheduling
      </a>
    </h6>
  </div>
</header>
`;
